### PR TITLE
Adds blog for community projects

### PR DESCRIPTION
**Pull request by elfisher**
### Description
Adds a blog to announce community projects page. This PR depends on https://github.com/opensearch-project/project-website/pull/153

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2021-6-30-introducing-community-projects.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+authors: 
+  - elifish
+date: 2021-06-30
+categories:
+  - community
+twittercard:
+  description: "Many open source projects benefit from the ecosystem of software created on top of or for the project. As a first steps towards having a robust ecosystem, opensearch.org now has added a community projects page to highlight projects created for the OpenSearch community by the OpenSearch community... "
+---
+
+Many open source projects benefit from the ecosystem of software created on top of or for the project. As an example, the Node Package Manager (NPM) has been instrumental in helping JavaScript be more approachable. It hosts a variety of modules that solve problems people face during the development of JavaScript software. Users can quickly search for these modules install them and incorporate them directly into their JavaScript project. Having a similar ecosystem for OpenSearch will help all of us get more out of our own OpenSearch deployments. As a first step, I wanted to share that [opensearch.org](/) now has a [community projects page](/community_projects/). The goal of the community projects page is to highlight projects created for OpenSearch.
```
Comment (stockholmux) [inline _posts/2021-6-30-introducing-community-projects.md:13]: `As an example, the Node Package Manager (NPM) has been instrumental in helping JavaScript be more approachable.` <- Passive voice

`The goal of the community projects page is to highlight projects created for OpenSearch.` <- this is a bit awkward

File: _posts/2021-6-30-introducing-community-projects.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+![OpenSearch Community Projects Page](/assets/media/blog-images/2021-06-30-introducing-community-projects/screenshot.png){: .img-fluid}
+
+**What kinds of projects are appropriate to highlight?**
+As mentioned above, the goal of the community projects page is to highlight projects created for OpenSearch. Any project that that helps people use or get more value out of OpenSearch is appropriate for the page, regardless of the project license. Projects could be OpenSearch plugins, OpenSearch Dashboards plugins, client libraries, ingestion tools, and more. For example, the first two projects highlighted are the [OpenSearch Plugin Template](https://github.com/AmiStrn/opensearch-plugin-template-java) (by [AmiStrn](https://github.com/AmiStrn)) and the [Flattened Data Type plugin](https://github.com/aparo/opensearch-flattened-mapper-plugin) (by [aparo](https://github.com/aparo)). Each project has a *name*, *description*, *icon (optional)*, *license*, *owner*, and *download link (optional)*.
```
Comment (stockholmux) [inline _posts/2021-6-30-introducing-community-projects.md:18]: 'that that helps'

**Review by stockholmux — CHANGES_REQUESTED** [review]
Just a few grammar things.

File: _posts/2021-6-30-introducing-community-projects.md (hunk 2 above)
Comment (elfisher) [reply _posts/2021-6-30-introducing-community-projects.md]: fixed

File: _posts/2021-6-30-introducing-community-projects.md (hunk 1 above)
Comment (elfisher) [reply _posts/2021-6-30-introducing-community-projects.md]: reworded. Let me know if the new wording works.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**